### DISCUSSION TITLE
How to hide password in User Model in adonis 5

### DISCUSSION THREAD

**Discussion by nickngqs** [adonisjs/core#1647] [Help] [answered]
In Adonisjs 4.1, you can hide for fields in Model

```
static get hidden () {
    return ['password']
  }
```

How do I hide in Adonis js 5.0

**Comment by thetutlage** [comment, marked as answer]
Have you read the docs for v5?

**Reply by nickngqs** [reply]
Apologised for that, I tried searching for it, didn't expect the behaviour was abit different from v4.

Combed through the docs, and managed to find. The solution is serializeAs: null

```
export default class Post extends BaseModel {
  @column({ serializeAs: null })
  public slug: string
}
```

**Reply by thetutlage** [reply]
Yup. Since, we are using decorators. There is no point in maintaining another array of properties. :)
